paddle/fluid/pir/transforms/onednn/cpu_bfloat16_placement_pass.cc

```diff
@@ -97,9 +97,29 @@ class OneDNNBf16PlacementPattern : public pir::RewritePattern {
       }
     }
 
-    int i = 0;
-    for (auto& value : op->operands_source()) {
-      pir::Type type = op->operand_type(i++);
+    const std::vector<std::string> permitted_input_names = {
+        "x", "y", "input", "residual_param"};
```

Comment:
Also need residual_data. For fused_matmul, it names residual_data as residual_param.

Reply:
Added. By the way, don't you think we could unify this name instead? Not sure where does the difference come from though...